### ISSUE TITLE
New settings editor needs to provide information on settings being saved

### ISSUE THREAD

**Issue by kbuchsicf** [Microsoft/vscode#63931]
Some sort of indicator at the top of the editor should include a SAVE button or if they are auto-saved there should be a temporary indicator that they were just saved.

**Comment by roblourens** [comment]
There is actually a notification, the first time you edit a setting, that settings are saved automatically. Maybe you didn't notice it?

**Comment by kbuchsicf** [comment]
Oh, so maybe you are talking about the "Modified" indicator. I assumed that was only indicating that the current setting was modified from default. Indeed there are cases where there are some checkboxes that never get the "x" in the box (seems like I saw that under Textg Editor > Text Editor but I cannot repeat it at this time). For those checkboxes, when I clicked it once it said "Modified" (though no check in box). When I click it again, the "Modified" is removed.

There is a dirty indicator on the editor tabs, but is not used by the Settings editor. 

Maybe I'm missing it. Here is what VS Code looks like:

![screen shot 2018-11-28 at 11 02 14 am](https://user-images.githubusercontent.com/34281089/49168442-1ce1ae00-f2fd-11e8-8c93-7ab740d797e8.png)
